napari/layers/shapes/shapes.py

```diff
@@ -1591,7 +1591,7 @@ def _is_color_mapped(self, color):
 
         raise ValueError(
             trans._(
-                'face_color should be the name of a color, an array of colors, or the name of an property',
+                'face_color/edge_color should be the name of a color, an array of colors, or the name of a property',
```

Comment:
```suggestion
                'Should be the name of a color, an array of colors, or the name of a property',
```
per @willingc here https://github.com/napari/napari/issues/7722#issuecomment-2751365729
I think making it more generic feels safer.